tests/unit/helpers.js

```diff
@@ -509,20 +510,43 @@ export const fakeRouterLocation = (props = {}) => {
   };
 };
 
+// TODO: remove this alias and use `createFakeLocation()` only.
```

Comment:
It only got renamed, right? Another option is to keep the name `fakeRouterLocation` and file an issue to rename everything to `createFakeLocation` later.

Reply:
Yes, I filed https://github.com/mozilla/addons-frontend/issues/5736